docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "Set Up"
+sidebar_order: 10
+description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry insights and tools into your web app to help your team troubleshoot more effectively.."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert title="Note" level="warning">
+    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite for having page-aware panels, as it allows us to collect page-specific issues and feedback through transactions.
+</Alert>
+
+You need to complete two steps to get the toolbar rendered on the page:
+1. Add or dynamically inject `<script defer src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>` into your web app. This script uses the `defer` attribute so that it does not block document parsing. The script tag can be inserted at the bottom of the page so it doesn’t block other critical Javascript.
+2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar.
+    ```html
+    <html>
+    <head>...</head>
+    <body>
+        …
+        <script async src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+        <script>
+        window.SentryToolbar.init({ … });
+        </script>
+    </body>
+    </html>
+    ```
+
+### Unmounting The Toolbar
+
+If you have called `SentryToolbar.init({...})` to render the toolbar, but now want to manually remove or unmount it from the page, you can call the cleanup function that is returned from `init()`. This will unmount all the injected HTML and CSS. Login credentials will not be removed, so you can re-insert the toolbar and still be authenticated.
+```html
+<script defer src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+<script>
+  const unmountToolbar = window.SentryToolbar.init({ … });
+
+  // sometime later...
+  unmountToolbar();
+</script>
+```
+
+## Implement Feature Flag Adapter
+
+In order to integrate your Feature Flagging platform with the Dev Toolbar, you will need an Adapter that can read flag data from your provider, and also store and retrieve a list of overrides to apply to your local browser session.
+
+We plan to create adapters for OpenFeature and LaunchDarkly soon!
```

Comment:
If you want to include this you should have a discussion or issue on GitHub to link to and remove the !. Otherwise I would say remove this sentence until you do.